test/libsolidity/semanticTests/isoltestTesting/balance_without_balance.sol

```diff
@@ -0,0 +1,7 @@
+contract ClientReceipt {
+}
+// ====
+// compileViaYul: also
+// ----
+// balance -> 0
+// balance: 0x0000000000000000000000000000000000000000 -> 0
```

Comment:
Does the wei/ether suffix works on the returned value?

Reply:
no